<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1205</width>
    <height>835</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>16</height>
       </size>
      </property>
      <widget class="QWidget" name="input_tab">
       <attribute name="title">
        <string>ΕΙΣΑΓΩΓΗ</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QGroupBox" name="expenseTableGroup">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QGroupBox" name="filtersGroup">
             <property name="minimumSize">
              <size>
               <width>800</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Calibri</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item alignment="Qt::AlignRight">
               <widget class="QLabel" name="dateLbl">
                <property name="minimumSize">
                 <size>
                  <width>60</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Calibri</family>
                  <pointsize>12</pointsize>
                  <kerning>true</kerning>
                 </font>
                </property>
                <property name="text">
                 <string>ΗΜΕΡΟΜΗΝΙΑ:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="dateFilter">
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>25</height>
                 </size>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignRight">
               <widget class="QLabel" name="userLbl">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Calibri</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>ΧΡΗΣΤΗΣ:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="userFilter">
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>25</height>
                 </size>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignRight">
               <widget class="QLabel" name="categoryLbl">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Calibri</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>ΚΑΤΗΓΟΡΙΑ:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="categoryFilter">
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>25</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QTableView" name="expenseTable">
             <property name="font">
              <font>
               <family>Calibri</family>
               <pointsize>16</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="verticalGroupBox">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QGroupBox" name="inputGroup">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>200</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>250</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Calibri</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <property name="leftMargin">
               <number>9</number>
              </property>
              <item row="3" column="0" alignment="Qt::AlignRight">
               <widget class="QLabel" name="categoryInputlbl">
                <property name="text">
                 <string>ΚΑΤΗΓΟΡΙΑ:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QDateEdit" name="dateInput">
                <property name="calendarPopup">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="4" column="0" colspan="2">
               <widget class="QPushButton" name="addExpenseBtn">
                <property name="font">
                 <font>
                  <family>Calibri</family>
                  <pointsize>14</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>ΠΡΟΣΘΗΚΗ</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0" alignment="Qt::AlignRight">
               <widget class="QLabel" name="valueInputLbl">
                <property name="minimumSize">
                 <size>
                  <width>40</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Calibri</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>ΠΟΣΟ:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1" alignment="Qt::AlignVCenter">
               <widget class="QLineEdit" name="valueInput">
                <property name="minimumSize">
                 <size>
                  <width>130</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Calibri</family>
                  <pointsize>16</pointsize>
                 </font>
                </property>
               </widget>
              </item>
              <item row="2" column="0" alignment="Qt::AlignRight">
               <widget class="QLabel" name="userInputLbl">
                <property name="text">
                 <string>ΧΡΗΣΤΗΣ:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1" alignment="Qt::AlignVCenter">
               <widget class="QComboBox" name="categoryInput"/>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="userInput"/>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="dateInputLbl">
                <property name="text">
                 <string>ΗΜΕΡΟΜΗΝΙΑ:</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="displaySelection">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>200</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1205</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
